src/test/java/seedu/address/model/studyspot/StudySpotTest.java

```diff
@@ -93,4 +95,12 @@ public void equals() {
         editedStarbucks = new StudySpotBuilder(STARBUCKS).withTags(VALID_TAG_QUIET).build();
         assertFalse(STARBUCKS.equals(editedStarbucks));
```

Comment:
I think youre missing the test with getAmenities() == false, thats why that line has partial code cov